### PR TITLE
AGL navbar fixes & login_redirect

### PR DESCRIPTION
**Pull request by alex-latham**
## All Submissions:

* [X] Merge the target branch into the PR branch. Fix any conflicts that might appear.
* [X] Have you checked to ensure that other pull request are approved for merge [Pull Requests](../../../pulls)?
* [X] What stories does the PR include
  - No stories, just making sure the navbar displays only what it should
### New Feature Submissions:

* [X] Have you added an explanation of what your changes do?
* [X] Have you lint your code locally prior to submission?

### Testing:

* [] Have you written new tests for your core changes, as applicable?
* [X] Have you successfully ran tests with your changes locally?
* [X] All new and existing tests passed.

**Test coverage** = 100%
--------------------------------------------------------------------------------
### Change Log
* Simplify SessionsController#new and #create to use login_redirect helper method (made during call with Cory). This makes the code more DRY.
* Reorder navbar links to follow conventional shopping sites
* Profile link in navbar does not display for visitors anymore
* Logout link in navbar does not display if a user is logged in anymore
* Bonus: unrestricts access to /profile (all registered users have a profile)
* Second Bonus: updates DTR

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -33,4 +25,12 @@ def destroy
     flash[:notice] = "You have successfully logged out!"
     redirect_to "/"
   end
-end
+
+  private
+
+  def login_redirect
+    redirect_to admin_path if current_admin?
```
Comment (danielpselph) [inline app/controllers/sessions_controller.rb:32]: Great work on refactoring the redirect paths

**Review by danielpselph — APPROVED** [review]
Effective work refactoring login redirect paths

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (Mariana-21) [reply app/controllers/sessions_controller.rb:32]: I agree great refactor!

**Review by Mariana-21 — APPROVED** [review]
Great Refactor on everything!

**Pull request merged**